vagrant.rb

```diff
@@ -412,19 +414,23 @@ def provisioning_command(machine, host_name, net, machines)
     end
     command += "/vagrant/scripts/cleanbox.sh /vagrant\n"
     command += "/vagrant/scripts/network.sh #{net_prefix} #{first_ip} \"#{host_list}\"\n"
-    command += "export DOWNLOAD_USER=#{ENV['DOWNLOAD_USER']} DOWNLOAD_PASSWORD=#{ENV['DOWNLOAD_PASSWORD']}\n"
     unless machine.key?('provision') then
+      network = net.to_s + "/" + net.prefix.to_s
```

Comment:
net is defined as: net = IPAddr.new ip + "/24", won't this produce 2 prefixes?
like 192.168.41.2/24/24?

Reply:
to_s only returns he IP part